COC_PyBay.md

```diff
@@ -0,0 +1,76 @@
+ - Meetups
+ - Workshops
+ - Presentation materials used in talks or sessions
+ - Slack
+ - Mailing lists
+ - GitHub
+ - Social media hashtag
+ - meetup.com discussion boards
+ - Any other forums created by the which the community uses for communication.
+
+In addition, violations of this code outside these spaces may affect a person's ability to participate within them.
+Speakers, presenters, and open space organizers
+
+Speakers, presenters, and those organizing open spaces or similar events are expected to uphold the community guidelines. Please let a member of staff know immediately if there are any issues during one of these events. Presentations, open spaces, and similar events will generally not be stopped for one-time gaffes or minor problems, although a staff member will speak to the presenter afterward and we will include any incidents in our transparency report. However, staff will take immediate action to politely and calmly stop any presentation or event that repeatedly or seriously violates our code of conduct policy.
+
+###Sponsors, affiliates, and exhibitors
+
+When you sponsor a PyBay event, we welcome you as a member of our community, and we expect you to be respectful to the community you operate within.
+
+All exhibitors in the expo hall, sponsor or vendor booths, or similar activities are also subject to the code of conduct. In particular, exhibitors should not use sexualized images, activities, or other material. Booth staff (including volunteers) must not use sexualized clothing/uniforms/costumes, or otherwise create a sexualized environment.
+
+In addition, sponsors and affiliates of conference, meetups, and online activities should not employ aggressive recruiting techniques, invasive marketing behavior, or similar actions towards community members. In case of violations, sponsors might be sanctioned and expelled from the event or activity with no return of the sponsorship contribution.
+
+###Weapons Policy
+
+No weapons are allowed at conference venues, including but not limited to explosives (including fireworks), guns, and large knives such as those used for hunting or display, as well as any other item used for the purpose of causing injury or harm to others. Anyone seen in possession of one of these items will be asked to leave immediately, and will only be allowed to return without the weapon.
+
+Attendees are further expected to comply with all state and local laws on this matter.
+
+###Young attendees
+
+Attendees of all ages are welcome at PyBay! Attendees 15 years of age and under are welcome to attend with a legal guardian. Both people should be registered for the conference and have badges. The legal guardian should be with the child at all times and the child should never be left alone at the conference.
+
+##What to do in case of violations
+
+If you believe that someone is violating the Code of Conduct during one of our events, please contact the CoC team immediately by emailing us at conduct@pybay.com.
```

Comment:
Can we make this into a Markdown link? `[conduct@pybay.com](mailto:conduct@pybay.com)`? Most things render it as a clickable e-mail but not all, and probably best to optimize usability of the reporting link.

Reply:
agreed.